crates/core/src/octree/build/mod.rs

```diff
@@ -181,13 +183,15 @@ impl Octree {
                 let geometry_first_node_in_level =
                     geometry_level_start_indices[octree_level as usize];
 
-                self.builder.allocate_nodes_pass.run(
-                    &self.geometry_data.voxel_data, // TODO: Pass in the actual number of nodes
-                    &self.textures,
+                let allocate_nodes_input = AllocateNodesInput {
+                    voxel_data: self.border_data.voxel_data.clone(),
```

Comment:
Changed geometry_data to border_data, although it makes sense it can be a breaking change (will test before merging)